Makefile

```diff
@@ -65,17 +66,37 @@ lint: ## Lint go source code
 .PHONY: lint
 
 # --- frontend -----------------------------------------------------------------
-frontend: | $(O) ## Build frontend, typically iterate with npm and inside frontend
+frontend: tiny wasm-exe | $(O) ## Build frontend, typically iterate with npm and inside frontend
 	rm -rf $(O)/public
 	cp -r frontend $(O)/public
+	rm -f $(O)/public/evy.wasm
```

Comment:
i think you can probably get rid of this `rm` and the next. Use `cp -f` instead.

Reply:
done.